radioapp/main.py

```diff
@@ -2,4 +2,5 @@
 Entry point for radioApp
 """
 from ui import ui_manager
-manager = ui_manager.UI()
\ No newline at end of file
+manager = ui_manager.UIManager()
+manager.main_loop()
```

Comment:
This call to `main_loop` needs to moved into a block like:

    if __name__ == "__main__":
        manager = ui_manager.UIManager()
        manager.main_loop()

Without this, the UI will start when main is imported. Should we be importing main? Probably not, but it's still an awful surprise for a window to pop up because of an import